src/utils.ts

```diff
@@ -15,6 +16,47 @@ export enum AcceptedSemverReleaseTypes {
   Patch = 'patch',
 }
 
+/**
+ * The different monorepo package version synchronization strategies employed
+ * by this Action.
+ */
+export enum VersionSynchronizationStrategies {
+  /**
+   * All packages will be updated to the new version, and the version of every
+   * monorepo package will be updated to the new version wherever it appears as
+   * a dependency.
+   *
+   * The published versions of all monorepo packages are "fixed".
+   *
+   * This is the default behavior _if_ the release is a new major version.
+   */
+  fixed = 'fixed',
+
+  /**
+   * Changed packages and their dependents in the monorepo will be updated to
+   * the new version, and the version of every monorepo package will be updated
+   * to the new version wherever it appears as a dependency.
+   *
+   * The published versions of all monorepo packages will reflect their
+   * dependency relationships.
+   *
+   * This is the default behavior _unless_ the release is a new major version.
+   */
+  transitive = 'transitive',
+
+  /**
```

Comment:
I've read this a few times and it's a little unclear to me what the difference between `transitive` and `independent` would be. The docs for `transitive` say:

> The published versions of all monorepo packages will reflect their dependency relationships.

whereas `independent` says:

> The published versions of all monorepo packages are "independent".

Is the intent here that `independent` would be the most selective version in the sense that if the root version gets bumped, only changed packages would get bumped, but their dependents would be untouched? If that's the case, then I am not seeing that reflected in the code. I've made a note within `shouldUpdateMonorepoPackage` in `package-operations.ts` that I believe should be changed to accommodate this.

Alternatively, if this is not something we need right now, then may I recommend that we drop `independent` for now and add it back later if we need it?